apidoc/validate.py

```diff
@@ -1,623 +0,0 @@
-#!/usr/bin/env python
```

Comment:
We should not remove that particular script, yet. It has some syntax-validations that the node-one does not currently have. I usually use both to scan changes, so I could setup a table of missing checks if necessary.